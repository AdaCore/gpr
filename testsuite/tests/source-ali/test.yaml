description:
    Test the ALI source info parser
driver: build_and_run
project_file: source_ali
main: main
