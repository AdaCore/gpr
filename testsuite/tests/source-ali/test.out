Project: Source_ALI
 > li/src/api.adb     language: Ada    Kind: S_BODY   unit: Api
 > li/src/api.ads     language: Ada    Kind: S_SPEC   unit: Api
 > li/main.adb        language: Ada    Kind: S_BODY_ONLY   unit: main
                                       deps: main.ali
 > li/src/pck.adb     language: Ada    Kind: S_BODY   unit: Pck
 > li/src/pck.ads     language: Ada    Kind: S_SPEC   unit: Pck
 > li/src/code.ada    language: Ada    Kind: S_SPEC_ONLY   unit: U1
                                       deps: code~1.ali
                                       Kind: S_SPEC   unit: U2
                                       Kind: S_BODY   unit: U2
                                       deps: code~3.ali
                                       Kind: S_BODY_ONLY   unit: U3
                                       deps: code~4.ali
Project: Source_ALI
 > li/src/api.adb     language: Ada    Kind: S_BODY   unit: Api
 > li/src/api.ads     language: Ada    Kind: S_SPEC   unit: Api
 > li/main.adb        language: Ada    Kind: S_BODY_ONLY   unit: Main
 > li/src/pck.adb     language: Ada    Kind: S_BODY   unit: Pck
 > li/src/pck.ads     language: Ada    Kind: S_SPEC   unit: Pck
 > li/src/code.ada    language: Ada    Kind: S_SPEC_ONLY   unit: U1
                                       Kind: S_SPEC   unit: U2
                                       Kind: S_BODY   unit: U2
                                       Kind: S_BODY_ONLY   unit: U3
